### PR TITLE
Add proposal to implement #1547, Concierge Impersonation Proxy | External Certificate Management

### PR DESCRIPTION
**Pull request by joshuatcasey**
Add proposal to implement #1547, Concierge Impersonation Proxy | External Certificate Management

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
👀

File: proposals/1547_impersonation-proxy-external-certs/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  manual interventions:
+    * Remove the new tls configuration block from the CredentialIssuer
+    * Clean up existing external CA/cert secret objects, and prevent their regeneration
+    * Regenerate and distribute a new kubeconfig for that cluster
+
+#### Tests
+
+No new integration tests, since the verification likely requires multiple components not typically installed during
+integration tests
+such as cert-manager and contour.
```
Comment (cfryanr) [inline proposals/1547_impersonation-proxy-external-certs/README.md:102]: We can have integration tests which generate their own CA and TLS certs using golang helpers (we already have these helpers, e.g. for generating Supervisor certs). Although this feature could be used with an ingress system like Contour, we shouldn't need that for tests. It should be possible to add new tests in `test/integration/concierge_impersonation_proxy_test.go`, which already configures/reconfigures the impersonation proxy via the CredentialIssuer CR.

Comment (joshuatcasey) [reply proposals/1547_impersonation-proxy-external-certs/README.md]: Updated to reflect that we *will* write integration tests

Comment (cfryanr) [reply proposals/1547_impersonation-proxy-external-certs/README.md]: I think you forgot to update the section called "Tests".

**Review by cfryanr — APPROVED** [review]

**Pull request merged**